accountCredentials.md

```diff
@@ -20,3 +13,37 @@ Specific information for finding your credentials can be found [here](https://su
 | `userId`                               | Your Bandwidth Account ID. The `userId` is used as part of the url to make API requests. This may be the same value as your accountId. <br> EX: `https://messaging.bandwidth.com/api/v2/users/{userId}/messages` <br><br> ⚠️ This **IS NOT** the same as your log in! ⚠️                                                                                |  `u-sd89g93epa2nb4euazd83kv8` OR `920012` |
 | `apiToken`                             | Your unique api token.  The `apiToken` is used as the **username** to authenticate to the API as part of the basic auth scheme                                                                                                                                                                       |  `f12a9edeed04ecd21b303c6f1f9f0831a1482f7f3c59199e`          |
 | `apiSecret`                            | Your unique api secret.  The `apiSecret` is used as the **password** to authenticate to the API as part of the basic auth scheme                                                                                                                                                                     |  `j54935lddasl837592356aasdf8359hlo3` |
+| `applicationId` | The ID of your messaging application | `532qd-fk5odk5-dlslka40-l5k3lsdmc` |
+
+## Authentication on each API
+
+### Voice
+
+| Credential Name  | How it is used |
+|:---------------------------------------|:-------------------|
+| `userId`                             | Used as a URL parameter for the API endpoint `https://voice.bandwidth.com/v2/account/{userId}`   |
```

Comment:
Is this the actual v2 voice URL Structure? I thought userId was scoped to messaging.

Reply:
I think it should be `accountId`, I'll update it real quick

Reply:
(Based on https://s3.amazonaws.com/slingshot-docs/methods/restApi.html)

Reply:
Fixed